samples/dotnet/kernel-syntax-examples/Example12_Planning.cs

```diff
@@ -121,6 +123,61 @@ private static async Task BookSamplesAsync()
         _ = await ExecutePlanAsync(kernel, planner, originalPlan);
     }
 
+    private static async Task MemorySampleAsync()
+    {
+        Console.WriteLine("======== Planning - Create and Execute Plan using Memory ========");
+
+        var memoryStorage = new VolatileMemoryStore();
+        var kernel = new KernelBuilder()
+            .WithLogger(ConsoleLogger.Log)
+            .Configure(
+                config =>
+                {
+                    config.AddAzureOpenAICompletionBackend(
+                        Env.Var("AZURE_OPENAI_DEPLOYMENT_LABEL"),
+                        Env.Var("AZURE_OPENAI_DEPLOYMENT_NAME"),
+                        Env.Var("AZURE_OPENAI_ENDPOINT"),
+                        Env.Var("AZURE_OPENAI_KEY"));
+
+                    config.AddAzureOpenAIEmbeddingsBackend(
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_DEPLOYMENT_LABEL"),
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_DEPLOYMENT_NAME"),
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_ENDPOINT"),
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_KEY"));
+                })
+            .WithMemoryStorage(memoryStorage)
```

Comment:
tiny nit: if you aren't doing anything with memoryStorage, you can use .WithMemoryStorage(new VolatileMemoryStore())